ouroboros-consensus/src/Ouroboros/Consensus/Util/CBOR/Simple.hs

```diff
@@ -0,0 +1,29 @@
+module Ouroboros.Consensus.Util.CBOR.Simple (
+    -- * Future-proofing
+    versionZeroProductFromCBOR
+  , versionZeroProductToCBOR
+  ) where
+
+import qualified Codec.CBOR.Decoding as CBOR
+import qualified Codec.CBOR.Encoding as CBOR
+import           Control.Monad (unless)
+
+{-------------------------------------------------------------------------------
+  Serialization aliases
+-------------------------------------------------------------------------------}
+
+-- | Simple codec to use since there's no version history to deal with and
```

Comment:
We should probably document what the parameters mean. In particular `arity` seems to be important.

Reply:
What is the "version history" in this context? Do we have uses of this version history that we could refer to?

Reply:
This module is gone from this PR now